lms/djangoapps/courseware/tests/test_course_info.py

```diff
@@ -216,15 +216,15 @@ def test_info_title(self):
         )
         self.assertIn(
             expected_title,
-            content('h1.page-title').contents()
+            content('.page-title').contents()[0]
```

Comment:
Yes, with the addition of the div within the header, we now have multiple items within .page-title's contents.  The other option is to add the new subtitle div to the expected title.